test/v1/fixtures/auth/AuthFixture.scala

```diff
@@ -138,7 +138,10 @@ object AuthFixture {
       data.credentialRole),
       None),
       data.credentialStrength),
-      data.loginTimes
+      data.loginTimes),
+      Some(data.itmpName)),
+      data.itmpDateOfBirth),
+      Some(data.itmpAddress)
     )
 
   val userDetails: (AffinityGroup, AgentInformation) => UserDetails = (Individual, agentInformation) => UserDetails(userType = Individual.toString,
```

Comment:
For the test maybe set at least one of the fields inside `itmpName` and `itmpAddress`, (and also `itmpDateOfBirth`) in `userDetails` and `orgIdentityData`.

This then proves that it is not just using the dummy/empty values in the service.

Reply:
Updated